jwst/lib/tests/test_dispaxis.py

```diff
@@ -0,0 +1,85 @@
+"""
+Test for dispaxis
+"""
+
+from .. import dispaxis
+
+def test_dispaxis_1():
+    value = dispaxis.get_dispersion_direction("FGS_IMAGE", "junk", "junk",
+                                              "junk")
+    assert value == 0
```

Comment:
Need to update the "0" to "None". Same for other instances that used to be zero.